symphony-bdk-core/src/main/java/com/symphony/bdk/core/client/ApiClientFactory.java

```diff
@@ -192,4 +200,26 @@ private void configureProxy(BdkProxyConfig proxyConfig, ApiClientBuilder apiClie
           .withProxyCredentials(proxyConfig.getUsername(), proxyConfig.getPassword());
     }
   }
+
+  @API(status = API.Status.EXPERIMENTAL)
```

Comment:
I don't know exactly which annotation I should use here, but if I don't use any the unit test will fail with this error:
```
com.symphony.bdk.core.arch.CoreArchitectureTest > classes_should_be_annotated_with_api_guardian FAILED
    java.lang.AssertionError at CoreArchitectureTest.java:5
```

Reply:
Internal I guess

Reply:
got it thanks!